src/frontend/komponenter/ManuellJournalføring/Dokumenter.tsx

```diff
@@ -0,0 +1,43 @@
+import { Journalstatus, RessursStatus } from '@navikt/familie-typer';
+import { AlertStripeAdvarsel } from 'nav-frontend-alertstriper';
+import React from 'react';
+import { useManuellJournalføring } from '../../context/ManuellJournalføringContext';
+import { DokumentVelger } from './DokumentVelger';
+
+export const Dokumenter: React.FC = () => {
+    const { dataForManuellJournalføring, valgtDokumentId } = useManuellJournalføring();
+    switch (dataForManuellJournalføring.status) {
+        case RessursStatus.SUKSESS:
+            return (
+                <div>
+                    {dataForManuellJournalføring.data.journalpost.journalstatus ===
+                    Journalstatus.MOTTATT ? (
+                        !dataForManuellJournalføring.data.journalpost.dokumenter ||
+                        dataForManuellJournalføring.data.journalpost.dokumenter.length === 0 ? (
```

Comment:
```suggestion
                        !dataForManuellJournalføring.data.journalpost.dokumenter?.length ? (
```

Jeg er ikke så stødig her, men tenkte kanskje det kunne finnes en mindre verbos måte å skrive det og fant[ dette](https://stackoverflow.com/questions/24403732/how-to-check-if-array-is-empty-or-does-not-exist). Vet ikke om det ble mer leselig, da 🙈

Reply:
We have changed it, but are not sure if this is what you suggested. Please have a look:)